electra/the fun part/fun.c

```diff
@@ -283,7 +283,7 @@ do { \
         unlink("/bootstrap/jailbreakd");
         
         rv = posix_spawn(&pd, tar, NULL, NULL, (char **)&(const char*[]){ tar, "-xpf", progname("basebinaries.tar"), "-C", "/" BOOTSTRAP_PREFIX, NULL }, NULL);
-        waitpid(pd, NULL, 0);
+        waitpid(pd, &rv, 0);
```

Comment:
what's the point of getting status if it's not used

Reply:
I was going to use it, forgot to remove this too